packages/components/button/src/Button.tsx

```diff
@@ -1,21 +1,21 @@
 import type { ButtonProps } from './button.types';
 import { ButtonStyled } from './button.styles';
+import theme from '../../../react/src/tokens/themes/theme';
 
 /**
  * Primary UI component for user interaction
  */
 const Button = ({
   primary = false,
   size = 'small',
```

Comment:
if you remove the backgroundColor, the user cannot choose the button color

Reply:
I agree with you, however, this button component have to be rebuilt, I have only use this color approach to test the type implementation.